lib_xud/src/core/XUD_Main.xc

```diff
@@ -487,7 +505,7 @@ int XUD_Main(chanend c_ep_out[], int noEpOut,
 
     for(int i = 0; i < USB_MAX_NUM_EP_IN; i++)
     {
-        handshakeTable_IN[i] = USB_PIDn_NAK;
+        handshakeTable_IN[i] = 0;//USB_PIDn_NAK;
```

Comment:
This comment is rather confusing. I'm guessing it's just not USB_PIDn_NAK anymore, and the comment should be removed?

Reply:
Fixed